src/plugins/headers.jl

```diff
@@ -0,0 +1,161 @@
+#  Copyright 2018, Oscar Dowson.
+#  This Source Code Form is subject to the terms of the Mozilla Public License,
+#  v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#  obtain one at http://mozilla.org/MPL/2.0/.
+
+# ================================ risk_measures ============================= #
+
+"""
+    AbstractRiskMeasure
+
+The abstract type for the risk measure interface.
+
+You need to define the following methods:
+ - Kokako.adjust_probability
+"""
+abstract type AbstractRiskMeasure end
+
+"""
+    adjust_probability(measure::Expectation
+                       risk_adjusted_probability::Vector{Float64},
+                       original_probability::Vector{Float64},
+                       noise_support::Vector{Noise{T}},
+                       objective_realizations::Vector{Float64},
+                       is_minimization::Bool) where T
+"""
+function adjust_probability end
```

Comment:
Do you plan to use quantization to reduce noises' support sizes?

Reply:
Care to elaborate on this? What do you mean?

Reply:
You currently use this function to build probability weights adjusted to a risk measure. I was wondering if we could also adjust the probability weights to decrease the size of the discrete support so as to improve numerical efficiency. You may have read this idea in [this article, p.3](https://pdfs.semanticscholar.org/6a29/e7bf2ffbe3aa010eb097e57460b5b06f10c8.pdf). 
This [other](https://arxiv.org/pdf/1701.04072.pdf) article (section 4, p.18) details also some good ideas but I didn't test them.